test/recipe/Baked.Test.Recipe.Service/CodingStyle/RichTransient/RichTransientWithData.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Baked.Test.CodingStyle.RichTransient;
+
+public class RichTransientWithData
+{
+    public RichTransientWithData With(string id)
+    {
+        Id = id;
+
+        return this;
+    }
+
+    public string Id { get; set; } = default!;
+
+    public object Method(object data) =>
```

Comment:
let's not accept and return an anonymous object here. use a string or anything strongly typed.